### PR TITLE
Spider

### PR DESCRIPTION
**Pull request by vpolisky**
The Spider module consists of:

- Django management command within the __cortex__ app (*/cortex/management/commands/feedspider.py*):

    Run command: `python manage.py feedspider`
    
    The command looks for sources which are not processed by spider and ready for crawling 
    and pushes their domain names into the __RabbitMQ__ message queue.
    
- __RabbitMQ__ message queue (currently running on my private server on *polisky.me* such that everybody can test a running version):

    *__vhost__: /worldbrain,__user__: worldbrain, __password__: worldbrain, __queue__: worldbarin*
    
    Domain names are queued in the queue and delivered to consumer as soon as available.
    
- A consumer deamon waiting for domain names in the queue (*/cortex/deamons/spider.py*):

    Run command: `./start_spider.sh` from within the *worldbrain/* directory
    
    The deamon is waiting for domain names in the message queue and asynchronously 
    starting a crawler process in a spawned separated process for each domain name. After crawling,
    the database is updated.

### CODE REVIEW

File: worldbrain/cortex/models.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ class Source(models.Model):
     domain_name = models.URLField()
     state = FSMField(default=SourceStates.PENDING.value, db_index=True)
     trusted_source = models.BooleanField(default=False)
+    processed_spider = models.BooleanField(default=False)
```
Comment (oliversauter) [inline worldbrain/cortex/models.py]: @hoshomoh and me talked about the different states of the domain here: https://github.com/WorldBrain/cortex/issues/40#issuecomment-247248803

Is this also reflected with your PR?

Comment (vpolisky) [inline worldbrain/cortex/models.py]: Thanks for the link! I tried to find infos about the source states and when they have been introduced, but I didn't find. This is partially reflected in my PR. I'll adjust this and remove the processed_spider field....

Comment (vpolisky) [inline worldbrain/cortex/models.py]: Hm, but what I don't understand now: My current version of the Source model is actually:
```
    domain_name = models.URLField()
    state = FSMField(default=SourceStates.PENDING.value, db_index=True)
    trusted_source = models.BooleanField(default=False)
    processed_spider = models.CharField(max_length=100, default='', blank=True)
    ready_for_crawling = models.BooleanField(default=False)
```
In your comment, you have `processed_spider = models.BooleanField`.  Seems to be another problem with my PR.

**Comment by hoshomoh** [comment]
There is also a merge conflict here @vpolisky. Merge master into your branch. If possible close this and make the PR again. Please squash your commit when making the PR.

File: worldbrain/cortex/models.py (hunk 1 above)
Comment (oliversauter) [inline worldbrain/cortex/models.py]: They have not really been introduced formally. We got to talk about this point last week, when I tried to understand how we can find out at what stage the domains are. 
Then I realised that it was part of @lwm's original design that got adopted by oshomo and tobi. 

Was a little bit of miscommunication in that regard.
Hope that doesn't cause too much extra work!

I think you already have the field "state" here, does that reflect the state of the process like described in #40 ?

**Pull request closed without merging**